packages/manager/modules/vrack/src/dashboard/vrack.constant.js

```diff
@@ -49,11 +49,85 @@ export const STATUS = {
 export const VRACK_DASHBOARD_TRACKING_PREFIX = 'vrack::dashboard';
 export const VRACK_ACTIONS_SUFFIX = 'action';
 
+/**
+ * groupes of services are used to display a sub list in the vrack available template
+ * @type {string[]}
+ */
+export const GROUPED_SERVICES_TYPES = [
+  'dedicatedCloudDatacenter',
+  'managedBareMetalDatacenter',
+  'dedicatedCloud',
+  'managedBareMetal',
+  'dedicatedServerInterface',
+];
+/**
+ * the services families are referenced in the vrack allowed services API
+ * this map is the base object to gather the data from the backend in each family
+ * https://api.ovh.com/console/#/vrack/%7BserviceName%7D/allowedServices~GET
+ * @type {{legacyVrack: null, dedicatedCloudDatacenter: null, ovhCloudConnect: null, dedicatedCloud: null, dedicatedServer: null, ip: null, dedicatedConnect: null, dedicatedServerInterface: null, cloudProject: null, ipLoadbalancing: null}}
+ */
+export const SERVICE_FAMILIES_MAP_FETCH = {
+  cloudProject: false,
+  dedicatedCloud: false,
+  dedicatedCloudDatacenter: false,
+  dedicatedConnect: false,
+  dedicatedServer: false,
+  dedicatedServerInterface: false,
+  ip: false,
+  ipLoadbalancing: false,
+  legacyVrack: false,
+  ovhCloudConnect: false,
+};
+export const SERVICE_FAMILIES_MAP = {
+  cloudProject: null,
+  dedicatedCloud: null,
+  dedicatedCloudDatacenter: null,
+  dedicatedConnect: null,
+  dedicatedServer: null,
+  dedicatedServerInterface: null,
+  ip: null,
+  ipLoadbalancing: null,
+  legacyVrack: null,
+  ovhCloudConnect: null,
+};
+/**
+ * only the keys of the service families map object
+ * @type {string[]}
+ */
+export const SERVICE_FAMILIES_LIST = Object.keys(SERVICE_FAMILIES_MAP);
+export const IP_SERVICE = 'ip';
+
+/**
+ * maps a type of serviec to a set of OVH UI icons
+ * @param serviceType
+ * @returns {{"ovh-font ovh-font-ip": boolean, "ovh-font ovh-font-iplb": boolean, "ovh-font ovh-font-server": boolean, "ovh-font ovh-font-vRack": boolean, "ovh-font ovh-font-dedicatedCloud": boolean, "ovh-font ovh-font-network": boolean, "ovh-font ovh-font-public-cloud": boolean, "oui-icon oui-icon-cloud-essential_concept": boolean}}
+ */
+export function serviceIconClassMapper(serviceType) {
+  return {
+    'ovh-font ovh-font-dedicatedCloud':
+      serviceType === 'dedicatedCloud' ||
+      serviceType === 'dedicatedCloudDatacenter',
+    'oui-icon oui-icon-cloud-essential_concept':
+      serviceType === 'managedBareMetal',
+    'ovh-font ovh-font-public-cloud': serviceType === 'cloudProject',
+    'ovh-font ovh-font-iplb': serviceType === 'ipLoadbalancing',
+    'ovh-font ovh-font-ip': serviceType === 'ip',
+    'ovh-font ovh-font-server':
+      serviceType === 'dedicatedServer' ||
+      serviceType === 'dedicatedServerInterface',
+    'ovh-font ovh-font-network': serviceType === 'dedicatedConnect',
+    'ovh-font ovh-font-vRack': serviceType === 'legacyVrack',
+  };
+}
```

Comment:
It shouldn't to have a function in constant file

Reply:
moved to controller